packages/p2p/src/components/dp2p-blocked/dp2p-blocked.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
 import { Icon, Text } from '@deriv/components';
-import Dp2pBlockedChecklist from './dp2p-blocked-checklist.jsx';
-import Dp2pBlockedDescription from './dp2p-blocked-description.jsx';
-import { Localize } from '../i18next';
+import Dp2pBlockedChecklist from './dp2p-blocked-checklist';
+import Dp2pBlockedDescription from './dp2p-blocked-description';
+import { Localize } from 'Components/i18next';
```

Comment:
can we also sort the imports? 
react import -> external libraries -> absolute imports -> relative imports alphabetically 
https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules